primitives/runtime/src/traits.rs

```diff
@@ -614,6 +614,8 @@ pub type HasherFor<B> = <HashFor<B> as Hash>::Hasher;
 pub type HashFor<B> = <<B as Block>::Header as Header>::Hashing;
 /// Extract the number type for a block.
 pub type NumberFor<B> = <<B as Block>::Header as Header>::Number;
+/// Extract the hash type for a block
+pub type BlockHashFor<B> = <B as Block>::Hash;
```

Comment:
@andresilva, and you said it is not needed

Reply:
Isn't that the same as `HashFor`?

Reply:
It is the same as `B::Hash` actually